.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "starcache"]
+	path = starcache
```

Comment:
better to put it under `be/src/thirdparty/starcache`. if we have multiple submodules in the future, it's more managable

Reply:
> better to put it under `be/src/thirdparty/starcache`. if we have multiple submodules in the future, it's more managable

OK